lib/workers/repository/update/pr/index.ts

```diff
@@ -281,9 +281,23 @@ export async function ensurePr(
   try {
     if (existingPr) {
       logger.debug('Processing existing PR');
+
+      const hasNotIgnoredReviewers = (pr: Pr): boolean => {
```

Comment:
no inline function, move to module scope

Reply:
Moved